curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md

```diff
@@ -83,6 +107,45 @@ const res1 = ['one|uno', '', 'three^^', 'four^|cuatro', ''];
 // TODO add more tests
 const testStr2 = 'a@&bcd&ef&&@@hi';
 const res2 = ['a&bcd', 'ef', '', '@hi'];
+
+const testStr3 = 'hello^|world^|how^are^you^|';
+const res3 = ['hello|world', 'how^are^you|'];
```

Comment:
This doesn't seem to be correct answer for the `'hello^|world^|how^are^you^|'` case. Could you check it again?